lib/pages/chat/events/message_content.dart

```diff
@@ -497,3 +486,63 @@ class _MessageImageBuilder extends StatelessWidget {
         matrixFile is MatrixImageFile;
   }
 }
+
+class _MessageVideoBuilder extends StatelessWidget {
+  final Event event;
+
+  final void Function(Event event) onFileTapped;
+
+  final Future<String> Function({required Event event})
+      handleDownloadVideoEvent;
+
+  const _MessageVideoBuilder({
+    required this.event,
+    required this.onFileTapped,
+    required this.handleDownloadVideoEvent,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final matrixFile = event.getMatrixFile();
+    DisplayImageInfo? displayImageInfo = Size(
+      MessageContentStyle.imageWidth(context),
+      MessageContentStyle.imageHeight(context),
+    ).getDisplayImageInfo(context);
+
+    final thumbnailSize = event.getOriginalSize();
```

Comment:
why thumbnail size is `originalSize`?

Reply:
it's resolution (width and height), let me change name